### ISSUE TITLE
Incorrect cursor behavior while editing the URL

### ISSUE THREAD

**Issue by allazakharenko** [tomlerendu/Websocket-King#7]
Cursor is constantly moved to the end of line while editing the URL
This bug appeared in the middle of January

It is reproducible both on Windows and Linux
![actual_result](https://user-images.githubusercontent.com/78749721/107251429-ff601100-6a3c-11eb-8229-067093f4cf44.gif)

**Comment by dishine** [comment]
Encountered the same problem

**Comment by allazakharenko** [comment]
At least someone feels my pain :D

пт, 5 мар. 2021 г. в 10:50, shinedi <notifications@github.com>:

**Comment by RoroTitiFR** [comment]
Same here, pretty annoying, but still the only problem I have on such great client overall!

**Comment by BallerIndustries** [comment]
Yeah, this is quite an annoying bug. I'm happy to help fix this if the source is available somewhere.

**Comment by MJLHThomassen-Sorama** [comment]
Same here, super annoying.

**Comment by MJLHThomassen-Sorama** [comment]
Any updates on this? Would also be happy to fix this.

**Comment by allazakharenko** [comment]
Hi! Nope, I guess, no one's going to get it fixed.I'm just a suffering passerby with any access to the code as well 😭Sent from my Huawei phone-------- Original message --------From: MJLHThomassen-Sorama ***@***.***>Date: Tue, Feb 15, 2022, 16:35To: tomlerendu/Websocket-King ***@***.***>Cc: allazakharenko ***@***.***>, Author ***@***.***>Subject: Re: [tomlerendu/Websocket-King] Incorrect cursor behavior while editing the URL (#7)
Any updates on this? Would also be happy to fix this.

**Comment by BallerIndustries** [comment]
We should band together and create a rival application called Websocket Queen

**Comment by allazakharenko** [comment]
LOL xD
I think they stop development as Postman claims to soon support websockets.

ср, 16 февр. 2022 г. в 15:37, Angus Cheng ***@***.***>:
